R/dots.R

```diff
@@ -1,23 +1,70 @@
-#' Extract dots with splicing semantics
+#' Extract tidy dots
 #'
-#' These functions evaluate all arguments contained in `...` and
-#' return them as a list. They both splice their arguments if they
-#' qualify for splicing. See [ll()] for information about splicing
-#' and below for the kind of arguments that qualify for splicing.
+#' @description
 #'
-#' `dots_list()` has _explicit splicing semantics_: it splices lists
-#' that are explicitly marked for [splicing][ll] with the
-#' [splice()] adjective. `dots_splice()` on the other hand has _list
-#' splicing semantics_: in addition to lists marked explicitly for
-#' splicing, [bare][is_bare_list] lists are spliced as well.
+#' `dots_list(...)` is equivalent to `list(...)` but provides tidy
+#' dots semantics:
+#'
+#' - You can splice other lists with the
+#'   [unquote-splice][quasiquotation] `!!!` operator.
+#'
+#' - You can unquote names by using the [unquote][quasiquotation]
+#'   operator `!!` on the left-hand side of `:=`.
+#'
+#' We call quasiquotation support in dots **tidy dots** semantics and
+#' functions taking dots with `dots_list()` tidy dots functions.
+#' Quasiquotation is an alternative to `do.call()` idioms and gives
+#' the users of your functions an uniform syntax to supply a variable
+#' number of arguments or a variable name.
+#'
+#' Note that while all tidy eval [quoting functions][quotation] have
+#' tidy dots semantics, not all tidy dots functions are quoting
+#' functions. `dots_list()` is for standard functions, not quoting
+#' functions.
+#'
+#'
+#' @section Comparison to tidy eval splicing:
+#'
+#' The `!!!` operator works differently in _standard_ functions taking
```

Comment:
This section needs to more concretely address _why_ a user should care about these differences